lib/RGBot.js

```diff
@@ -1852,7 +1906,18 @@ class RGBot extends EventEmitter {
         return result
     }
 
-
+    /**
+     * Pauses the bot to wait for the desired number of milliseconds. Use this
+     * function when you want the bot to wait for a certain amount of time, instead
+     * of a certain number of ticks (see `wait(ticks: number)`).
+     * @see wait
+     * @param {number} milliseconds The number of milliseconds to wait
+     * @return {Promise<void>}
+     */
+    async waitForMilliseconds(milliseconds) {
```

Comment:
Can we move this up in the file so it's right before or right after `waitForTicks`? The generated .md relies on the order that methods appear in the source code, so we need to keep these organized ourselves for now

Reply:
Fixed!